### ISSUE TITLE
About the --load_from

### ISSUE THREAD

**Issue by znxnx** [Chenrj233/LMEDR#5]
Hello, I would like to ask that during Evaluation, there seems to be no input parameter --load_from in the file evaluation_PersonaChat.py

**Comment by Chenrj233** [comment]
Yes, you are right, it should be corrected to `--model_checkpoint`
